.eslintrc.js

```diff
@@ -20,10 +22,21 @@ module.exports = {
   },
   globals: {
     // leaflet maps global
-    L: 1
+    L: 'writeable'
```

Comment:
I'm not sure what `writeable` does (the other 2 options were `readable` and `off`). The [changes to `eslint`](https://github.com/eslint/eslint/pull/11517/files#diff-5b7a32242c85597b4dc62430676113788637b44546e05f4c831cb3b593815d87L401) showed that the value of `1` would be `writeable` by default.